src/AndcultureCode.CSharp.Sitefinity.Core/Models/Services/ODataServiceResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using AndcultureCode.CSharp.Sitefinity.Core.Interfaces;
+using System.Collections.Generic;
+
+namespace AndcultureCode.CSharp.Sitefinity.Core.Models.Services
+{
+    public class ODataServiceGetResponse<T> where T : ISitefinityContentDto
```

Comment:
Should this file name be renamed to `ODataServiceGetResponse` to reflect this class name?